src/client/utils/seedIsInSample.js

```diff
@@ -0,0 +1,8 @@
+import seedRandom from 'seed-random';
+
+export function seedIsInSample (seed, percent) {
```

Comment:
should we have tests for this?

Reply:
i suppose it'd mostly be testing the implementation of `seed-random`

Reply:
My thoughts exactly!